src/main/java/uk/gov/pay/api/model/SettlementSummary.java

```diff
@@ -24,7 +24,7 @@ public SettlementSummary(String captureSubmitTime, String capturedDate) {
         this.capturedDate = capturedDate;
     }
 
-    @ApiModelProperty(value = "Date and time capture request has been submitted", example = "2016-01-21T17:15:00Z")
+    @ApiModelProperty(value = "Date and time capture request has been submitted (may be null if capture request was not acknowledged by payment gateway)", example = "2016-01-21T17:15:00Z")
```

Comment:
This covers the case where the capture request times out but they then send us a notification saying it was captured, right? Perhaps “may be null if capture request was not immediately acknowledged by payment gateway” (note “immediately”) would be a little clearer.